internal/pkg/data/sql-datastore-translator.go

```diff
@@ -190,9 +194,14 @@ func (ds *sqlDatastoreTranslator) translatePrepared(input data.Node) (q string,
 			if sqlCallOp, ok := ds.callOps[op]; ok {
 				// Expected stack:  top -> [args..., call-op]
 				logging.LogForComponent("sqlDatastoreTranslator").Debugln("NEW FUNCTION CALL")
-				nextRel = sqlCallOp(ops[1:]...)
+				var callOpError error
+				nextRel, callOpError = sqlCallOp(ops[1:]...)
+				// Check for call operand error
+				if callOpError != nil {
+					return callOpError
+				}
 			} else {
-				logging.LogForComponent("sqlDatastoreTranslator").Panic(fmt.Sprintf("Operator [%s] is not supported!", op))
+				return errors.Errorf("Operator [%s] is not supported!", op)
```

Comment:
I would give some more information in the error message. Something like: "Probably a function X of your policy is not mapped by any config" or so in addition to this Error.

Reply:
Same question as above